### PR TITLE
My booking app PR#1

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, we created the booking application for luxurious cars:
- set the project backend and the frontend up
- add a navigation panel qhere the user can see links to:
- cars
"Reserve" form.
"My reservations".
"Add car (the link is visible to everybody).
"Delete car (the link is visible to everybody).
On the main page, the user can see a [list of cars
4. When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description (skip the "Rotate image" button).
In the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).
Details

When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.

- We made the app responsive, creating both mobile and desktop versions.

### CODE REVIEW

**Review by thasipablo — APPROVED** [review]
## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Setup controllers correctly ✔️ 
- Setup models correctly ✔️ 

Cheers, and Happy coding!👏👏👏

File: README.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+<a name="#readme-top"></a>
```
Comment (Meltrust) [inline README.md:1]: - [ ] I couldn't find a working backend app for this project, so much of the functionality is not present at the time of this review, as you kindly noted in your review request.

note: this means I couldn't test the core functionality of the app or the code itself.  These items will be taken a look at in the next review.

Comment (Meltrust) [inline README.md:1]: - [ ] This is the frontend app, however, much of the readme instructions in this repo are for the Rails backend.  Kindly make it so the frontend repo has the specific instructions to setup and run the frontend app correctly. 👍

Comment (Meltrust) [inline README.md:1]: - [ ] In order to get into the frontend app, the user needs to at least type their username.  That way, different users can test the app and have their own reservations/bookings. 👍

Comment (Meltrust) [inline README.md:1]: - [ ] Your frontend app code has a lot of commented code across several files.  Kindly remove it in order to preserve the repo's cleanliness. 👍

Comment (Meltrust) [inline README.md:1]: - [ ] On your readme, these links won't work as they just lead to the repo's main page:

image


Kindly fix this links so they lead to what is specified in the labels. 👍

Comment (Meltrust) [inline README.md:1]: - [ ] When you add the login page, kindly remember that there should be a way to log out. 👍

**Review by Meltrust — COMMENTED** [review]
_note: As requested by the student, this is a review for only the frontend app and their work in progress.   Since I couldn't find a working backend app to consume data, much of the frontend functionality is not present yet._

_note 2: This review was requested on a PR that has already been merged._


Hi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:


-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏



------

**Pull request merged**